Dockerfile

```diff
@@ -1,47 +1,58 @@
 # syntax = docker/dockerfile:1.0-experimental
 ARG VAGRANT_VERSION=2.2.19
 
-
-FROM ubuntu:bionic as base
```

Comment:
The choice of `ubuntu:bionic` was to ensure that the client library wouldn't be trying to use newer features when talking to the service side when talking via the socket when run on likes of CentOS/RHEL 7 / Ubuntu 18.04. While it might be fine to switch debian flavour that uses a more recent libvirt (v7 vs v4) I think it might need more testing to check that it won't have issues when run against an older version of the libvirt server. That would need to be done before this PR could land, so perhaps better dealing with that as a separate PR where we can either have an additional set of images to have the newer libvirt if it doesn't work with the older distros or switch once confirmed that it will work.